srcpkgs/python-stem/template

```diff
@@ -3,28 +3,9 @@ pkgname=python-stem
 version=1.7.1
 revision=2
```

Comment:
Revbump

Reply:
Why would we revbump something that is meta(for removal)?

Reply:
Because the existing pkg in the xbps repo won't be updated